src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -70,27 +70,27 @@
       <Link>Lucene.Net.dll</Link>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="..\NuGetConsole.Host.PowerShell\Scripts\Add-WrapperMembers.ps1">
+    <Content Include="$(ArtifactsDirectory)\Scripts\Add-WrapperMembers.ps1">
```

Comment:
Wouldn't this lead to some warnings/errors if the project is not build? 

The artifacts directory is creating at build time.

Reply:
In Solution Explorer, the PowerShell script files are unopenable from the NuGet.VisualStudio.Client project until you build:

![image](https://user-images.githubusercontent.com/12734758/58658484-6d9a8980-82d5-11e9-8f1e-0195251341b5.png)

This is pretty benign though.  The files are openable from the NuGet.VisualStudio.Client project after you build, and they are always openable from their source project NuGetConsole.Host.PowerShell whether or not you have built.

Otherwise, I see no warnings or errors.